src/App.asm

```diff
@@ -1,6 +1,55 @@
 .data
+i : .word 0
+top_student : .word -1
+score: .word 0
+count :.word 0
+str1 : .asciiz "Enter total number of students: "
+str2: .asciiz "Enter student score: "
+str3: .asciiz "Highest score is "
+.globl main
 
+.text
 
-.globl main
+main :
+	li $t7,1
+	li $v0,4
+	la     $a0, str1
+	syscall
+	li $v0,5 
+	syscall
+	move $t0,$v0 # count
+	sw $t0,count
+	
+	lw $t1,i
+loop_start:
+ blt $t1,$t0, loopBody
```

Comment:
variables (`i` and `count`) used in while loop condition must be loaded from memory before such as:
```asm
...
lw  $t1, i
lw  $t0. count
blt $t1, t0, loopBody
...
```